include/json/value.h

```diff
@@ -539,17 +539,17 @@ Json::Value obj_value(Json::objectValue); // {}
   bool removeMember(const char* key, Value* removed);
   /** \brief Remove the named map member.
 
-      Update 'removed' iff removed.
+      Update 'removed' if removed.
```

Comment:
I don't think this is a typo. "iff" typically is math speak for "if and only if".

Reply:
Uh, I understand now. thanks a lot. I'll close this PR.